app/views/products/_modal.html.erb

```diff
@@ -0,0 +1,11 @@
+<div class="modal-header">
+  <button type="button" class="btn-close" data-dismiss="modal" aria-label="Close"></button>
+  <h4 class="modal-title" id="myModalLabel">Delete product?</h4>
+</div>
+<div class="modal-body">
+  Are you sure?
+</div>
+<div class="modal-footer">
+  <%= link_to 'Delete', product_path(@product), data: {method: :delete}, class: 'btn btn-danger' %>
+  <%= link_to 'Cancel', product_path(@product), class: 'btn btn-info' %>
```

Comment:
Cancel button can be simple button without link, in this case you will make a redirect and reload a page just for closing modal window which is redundant